packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -175,6 +175,7 @@ const pauseSession = (
     delete account.session.sessionDeadline;
     account.session.registeredUtxos = [];
     account.session.paused = true;
+    account.session.interrupted = !!payload.interrupted;
```

Comment:
nit, is `!!` required since type is optional?

Reply:
Removed `!!`.